### ISSUE TITLE
Timeout set to pod ready timeout 130000 - kind

### ISSUE THREAD

**Issue by rchojn** [eclipse-che/che#17431]
### Che version

 - [x] latest
 - [ ] nightly
 - [ ] other: please specify

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [x] other: (please specify)

### Installation method
   - [ ] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [x] my computer
       - [ ] Windows
       - [x] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

### Additional context

```
 ✅  Post installation checklist
    ❯ PostgreSQL pod bootstrap
      ✔ scheduling...done.
      ✔ downloading images...done.
      ✖ starting
        → ERR_TIMEOUT: Timeout set to pod ready timeout 130000
      Keycloak pod bootstrap
      Devfile registry pod bootstrap
      Plugin registry pod bootstrap
      Eclipse Che pod bootstrap
      Retrieving Eclipse Che server URL
      Eclipse Che status check
    Retrieving Keycloak admin credentials
    Retrieving Che self-signed CA certificate
    Show important messages
 ›   Error: Error: ERR_TIMEOUT: Timeout set to pod ready timeout 130000
 ›   Installation failed, check logs in '/tmp/chectl-logs/1594910965802'
```

I've got this error on my kind cluster. What could be the reason?

**Comment by tolusha** [comment]
@gardner-dev 
Pls, follow the bug template

**Comment by gardner-dev** [comment]
**Runtime**
- kind 

**Installation method**
- installation on kind instruction

@tolusha done

**Comment by l0rd** [comment]
@gardner-dev I would say that's a duplicate of https://github.com/eclipse/che/issues/14469

**Comment by tolusha** [comment]
@gardner-dev 
Could you provide logs, they are available at `/tmp/chectl-logs/1594910965802`

@l0rd 
I don't think so.
